packages/manager/apps/sap-features-hub/public/translations/installation/Messages_fr_FR.json

```
@@ -49,5 +49,18 @@
   "os_config_tooltip_firewall_server": "Autorise toutes les communications sur les serveurs d'application en provenance des autres serveurs de ce système SAP",
   "os_config_toggle_firewall_database": "Activer le firewall pour la base de données SAP HANA",
   "os_config_tooltip_firewall_database": "Autorise toutes les communications sur la base de données SAP HANA en provenance des autres serveurs de ce système SAP",
-  "os_config_cta": "Personnalisez vos machines virtuelles"
+  "os_config_cta": "Personnalisez vos machines virtuelles",
+  "enablement_title": "Activez nos fonctionnalités SAP optionnelles",
+  "enablement_subtitle": "Vous pouvez activer  et configurer nos fonctionnalités SAP optionnelles en fonction de vos besoins.",
+  "enablement_cta": "Afficher le récapitulatif",
+  "enablement_input_has_backup": "Agent de sauvegarde OVHcloud Backint pour les bases de données SAP HANA",
+  "enablement_input_id_container_tooltip": "Ce conteneur sera utilisé pour stocker vos sauvegardes DATA et LOG.",
+  "enablement_input_endpoint": "Endpoint",
+  "enablement_input_endpoint_helper": "L'endpoint est au format URL.",
+  "enablement_input_has_logs_ldp_ovh": "Logs SAP via la Logs Data Platform d'OVHcloud",
+  "enablement_input_has_logs_ldp_ovh_helper": "Réalise la configuration OS sur tous les serveurs du SID pour exporter les logs SAP (y compris \"Security Audit Log\") vers le Logstash précisé ci-dessous (nécessite des post-configurations manuelles supplémentaires.",
```

By the way, `nécessite des post-configurations manuelles supplémentaires` is a bit weird in French
```suggestion
  "enablement_input_has_logs_ldp_ovh_helper": "Réalise la configuration de l'OS sur tous les serveurs du SID pour exporter les logs SAP (y compris \"Security Audit Log\") vers le Logstash mentionné ci-dessous (nécessite des post-configurations manuelles supplémentaires).",
```